README.md

```diff
@@ -1,3 +1,160 @@
 The full documentation on Amido Stacks can be found [here](https://amido.github.io/stacks/).
 
-Java Spring boot Rest API with CQRS documentation can be found [here](https://amido.github.io/stacks/docs/workloads/azure/backend/java_cqrs/intro_java_cqrs).
\ No newline at end of file
+## Using the repository to build the Java Spring Boot Rest API application
+
+### Versioning
+
+This is version `1.0.0` of the `java-stacks-cqrs-events` project.
```

Comment:
Wouldn't this be a pain to maintain the correct version?

Reply:
You're right, I'll remove it or basically call it v2